File: Frontend/Private/gco-private/src/app/model/core/score.ts
import { IdMessage } from "../dto/id-message";
import { Apparatus } from "./apparatus";

export interface Score{
    id?: string;
    dScore: number;
    eScore: number;
    totalScore: number;
    apparatus: Apparatus
    competitionId: string;
    contestant: IdMessage;
    submitted: boolean;
}

File: Backend/scoring_service/core/domain/score.go
package domain

import "github.com/google/uuid"

type Score struct {
	ID            uuid.UUID
	Apparatus     Apparatus
	DScore        float32
	EScore        float32
	TotalScore    float32
	CompetitionID uuid.UUID
	Competition   Competition
	ContestantID  uuid.UUID
	Contestant    Contestant
	Submitted     bool
}
